### DISCUSSION TITLE
Fix Blackrock Depths - Upper City Spawning Point Please

### DISCUSSION THREAD

**Discussion by Mcadeez** [chromiecraft/chromiecraft#5502] [Suggestions]
Hello,

It's been over a year now playing on ChromieCraft's Progression Server and I ran into the issue where when queuing Random Dungeon in the Random Dungeon Finder, the group will often think it's a Blackrock Depths Prison run based on where we spawn. Upon killing the final boss for the Prison run, we discover the hard way that the instance is actually an Upper City run. 

Please fix the spawning point for the Upper City BRD runs. I hate getting into BRD Upper City runs and taking 2-4 hours to clear it because we don't have the Shadowforge Key, we spawn at the very beginning of the Prison runs, we have to explain the bug to newcomers, people haven't ran the content in years, etc. 

This is just a suggestion, but also a bug in the code for the game I suppose? We are into TBC now for content, but yet still having trouble with dungeons before Classic Raids.

Sincerely,
Mcadeez

**Comment by Tereneckla** [comment]
Both having the same spawn point sadly is not a bug.

**Reply by Mcadeez** [reply]
Hello Tereneckla.

Is there a way to add a portal(s) near the entrance when completing sections of the instance? 

For example, if the group wipes but we made it to the great hall with the respawning dwarves, could we have a portal to fast travel back to that room to save time? 

The main struggle I see in most groups that get queued into that instance is that the only source of revival is a resurrection spell from the healer or releasing and walking back to their body.

For example, if say the healer dies just before the final room with the emperor boss, then the group either: (1) waits until the healer returns, (2) travels back to the beginning to aid the healer in returning to the end of the instance. 

Would adding a portal near the beginning of the entrance be an option to consider? 

Looking forward to a response. 

Please and thank you. 

~ Mcadeez

**Reply by Tereneckla** [reply]
It is possible to make a custom NPC
If that is added is not for me to decide

**Comment by Gultask** [comment]
It is, indeed, not a bug, and since the majority of players were not running BRD at the time it was changed, we didn't put much effort in changing back, especially since it was also not a bug.

We do understand that is unfortunate, really.

Since Challenge Modes gave the lower brackets a resurgence of players, we decided to look further again into this and add it as a QoL change, as well as removing the requirement for having the Shadowforge Key to progress through the dungeon.

The changes are now live in CC as of today.
